3_javascript/javascript1/week1/future-age/futureAge.js

```diff
@@ -0,0 +1,7 @@
+function calculateAge(){
+    const yearOfBirth = document.getElementById("yearOfBirth").value;
+    const yearFuture = document.getElementById("yearFuture").value;
+    const p = document.getElementById("yourAge");
+    let yourAge = yearFuture - yearOfBirth;
+    p.innerHTML = `You will be ${yourAge} years old in ${yearFuture}`
+}
```

Comment:
If you want to make it more complicated 🤓  you can add checking what if year of birth and year of future user will remain as empty and click the button, maybe there can be string like _dont leave fields empty_ ? 😄